packages/core/content-releases/admin/src/pages/ReleasesPage/tests/Releases.test.tsx

```diff
@@ -5,6 +5,13 @@ import { IntlProvider } from 'react-intl';
 
 import { ReleasesPage } from '../Releases';
 
+jest.mock('@strapi/helper-plugin', () => ({
+  ...jest.requireActual('@strapi/helper-plugin'),
+  // eslint-disable-next-line
+  CheckPermissions: ({ children }: { children: JSX.Element}) => <div>{children}</div>,
+  CheckPagePermissions: ({ children }: { children: JSX.Element }) => <div>{children}</div>,
+}));
```

Comment:
To avoid this people do one of two things:
* export a `ProtectedPage` and `Page` function
* wrap the whole thing in the providers – `admin/admin/test/utils` is a good example of how you can do this.

from experience the less mocking the better your tests run

Reply:
Really thank you for these suggestions. I tried to follow the 2nd approach but I end with a lot of "wrap actions inside an act" errors on the test 🤔 

With the first one I can remove the CheckPagePermissions, but for now I still need to mock `CheckPermissions` because it's used inside the page 🤔

Reply:
Yeah the act warnings might be because you're not awaiting async loading - most likely from the useRBAC hook, if you can assert UI from that you should be able to avoid it.